apps/api/src/app/testing/rate-limiting.controller.ts

```diff
@@ -3,9 +3,13 @@ import { Controller, Get, UseGuards } from '@nestjs/common';
 import { ThrottlerCategory, ThrottlerCost } from '../rate-limiting/guards';
 import { UserAuthGuard } from '../auth/framework/user.auth.guard';
 import { ExternalApiAccessible } from '../auth/framework/external-api.decorator';
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { ApiKeyAuth } from '../shared/framework/swagger/api.key.security';
 
 @Controller('/rate-limiting')
 @UseGuards(UserAuthGuard)
+@ApiBearerAuth()
+@ApiKeyAuth()
 export class TestApiRateLimitController {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** I guess we can exclude these

Reply:
@rifont it says   @ExternalApiAccessible()
 @scopsy what to do here?